.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
     sourceType: 'module'
   },
   rules: {
-    semi: [2, "never"]
+    semi: [2, "never"],
+    'max-statements-per-line': 'off'
```

Comment:
Without updating this rule we get this linter error in `getGalleryRow`:

```
This line has 2 statements. Maximum allowed is 1  max-statements-per-line
```
I believe it's also an option to set it to 1 for a warning instead of an error, if we don't want to turn it off completely